chaoshub/cli.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+import io
+import json
+from logzero import logger
+
+import click
+
+__all__ = ["login"]
+
+from chaoshub.publish import publish_to_hub
+from chaoslib.settings import load_settings, save_settings
+from chaoslib.types import Settings
+
+
+@click.command()
+@click.argument('login_target')
```

Comment:
`login_target` may not be the happiest naming here, don't you think? 

Could also use the `help="...."` argument to be set to document that argument :)

Reply:
Done.